get-started/mac/first-steps-nuget.md

```diff
@@ -0,0 +1,84 @@
+---
+title: First Steps with NuGet
+page_title: macOS Getting Started Guide for Installing with Telerik NuGet Server and VS
+description: "Get started with Telerik UI for .NET MAUI and learn how to install the controls by using the Telerik NuGet Server with Visual Studio for macOS."
+tags: maui, dotnet maui, microsoft maui, telerik maui, nuget, ui for .net maui, macos, install
+slug: telerik-nuget-server-mac
+position: 2
+---
+
+# First Steps by Installing with the NuGet Package on macOS
+
+NuGet is a popular .NET package manager. Progress maintains the Telerik NuGet Feed for registered users and you can include the Telerik UI for .NET MAUI suite in your project as well as update to the latest available version from there.
+
+While installing Telerik UI for .NET MAUI with NuGet works both for Windows and MacOS machines, this tutorial describes how to get up and running with the library by downloading and installing the controls on macOS. Here is the step by step guide you have to follow:
+
+## Step 1: Set Up Your .NET MAUI Project
+
+Before you start with the installation of Telerik UI for .NET MAUI, make sure you have a running .NET MAUI application. For more information on the required steps and system requirements, refer to the [Microsoft .NET MAUI official documentation](https://docs.microsoft.com/en-us/dotnet/maui/get-started/installation).
+
+## Step 2: Download Telerik UI for .NET MAUI
+
+Telerik UI for .NET MAUI enables you to download the suite either from the Telerik UI for .NET MAUI product page or through your Telerik account. For the purposes of this tutorial, let's download the batch from your Telerik account:
+
+1. Log into your [Telerik Account](https://www.telerik.com/account/).
+
+1. Click the __Downloads__ tab.
+
+  ![Telerik UI for .NET MAUI Download tab in your account](../../images/download-tab.png)
```

Comment:
The images break the list, it looks like they need an additional space before the "!".
```suggestion
   ![Telerik UI for .NET MAUI Download tab in your account](../../images/download-tab.png)
```